.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-dist
\ No newline at end of file
+dist
+package-lock.json
```

Comment:
thanks for your PR, looks great. One thing I notice is that according to https://docs.npmjs.com/files/package-lock.json, we might want to commit this file